packit/command_handler.py

```diff
@@ -114,7 +114,7 @@ def run_command(
         )
         sandcastle.run()
         try:
-            logger.info("running command: %s", command)
+            logger.info(f"Running command: {' '.join(command)}")
```

Comment:
How about: 

```suggestion
            logger.info(f"Running command: {command!r}")
```

for simplicity's sake?

Reply:
I prefer the first here, as I consider 
`Running command: git describe --abbrev=0 --tags --match *`
more readable then
`Running command: ['git', 'describe', '--abbrev=0', '--tags', '--match', '*']`